### PR TITLE
Refactor node packaging

### PR DESCRIPTION
**Pull request by kaniini**
* Convert `nodejs` into a proper version stream, e.g. `apk add nodejs~19`, will pick nodejs 19, and so on.
* Drop bundled `npm`, replace it with a common npm across all versions that reflects the latest release.
* Add `nodejs-lts` version stream which contains node 16 and 18 at the moment.

### CODE REVIEW

File: npm/npmrc [hunk 1]
```diff
@@ -0,0 +1,253 @@
+<!DOCTYPE html>
```
Comment (ajayk) [inline npm/npmrc:1]: Probably this needs to be replaced ?

Comment (kaniini) [reply npm/npmrc]: Yeah, haha, oops.

File: nodejs-18.yaml [hunk 2]
```diff
@@ -1,13 +1,16 @@
 package:
   name: nodejs-18
   version: 18.17.0 # When bumping this, also bump the provides below!
-  epoch: 0
```
Comment (ajayk) [inline nodejs-18.yaml:4]: epoch should be set to 1 ?

File: nodejs-18.yaml [hunk 3]
```diff
@@ -1,13 +1,16 @@
...
+  epoch: 2
```
Comment (ajayk) [inline nodejs-18.yaml:4]:
```suggestion
  epoch: 1
```

**Review by ajayk — APPROVED** [review]

**Pull request merged**